### PR TITLE
Remove use of ts-loader

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: example/webpack.conf.js [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
     filename: './bundle.js'
   },
-  resolve: {
-    extensions: ['', '.ts', '.js']
-  },
-  bail: true,
   module: {
     loaders: [
-      { test: /\.ts$/, loader: 'ts-loader' },
       { test: /\.css$/, loader: 'style-loader!css-loader' },
```
Comment (sccolbert) [inline example/webpack.conf.js]: we don't need style loader for this example either

File: package.json [hunk 2]
```diff
@@ -34,13 +33,15 @@
   "scripts": {
     "clean": "rimraf lib && rimraf test/build && rimraf test/coverage",
     "clean:example": "rimraf example/bundle.js && rimraf example/node_modules/phosphor-keymap",
```
Comment (sccolbert) [inline package.json]: can you update this command to remove the build `js` files (bundle.js is no longer present)

File: test/webpack-cov.conf.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+module.exports = {
+  entry: './test/build/index.js',
+  output: {
+    filename: './test/build/coverage.js'
+  },
+  bail: true,
+  module: {
+    loaders: [
+      { test: /\.css$/, loader: 'style-loader!css-loader' },
```
Comment (sccolbert) [inline test/webpack-cov.conf.js]: style loader not needed

File: test/webpack.conf.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+module.exports = {
+  entry: './test/build/index.js',
+  output: {
+    filename: './test/build/bundle.js'
+  },
+  bail: true,
+  module: {
+    loaders: [
+      { test: /\.css$/, loader: 'style-loader!css-loader' },
```
Comment (sccolbert) [inline test/webpack.conf.js]: style loader not needed

File: package.json [hunk 5]
```diff
@@ -26,7 +26,6 @@
     "mocha": "^2.2.5",
     "rimraf": "^2.4.2",
     "style-loader": "^0.13.0",
```
Comment (sccolbert) [inline package.json]: style loader not needed

File: package.json (hunk 2 above)
Comment (blink1073) [inline package.json]: Bundle.js is still there, added index.js as well.

**Comment by blink1073** [comment]
Updated.

**Pull request merged**